src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/sort/SortClause.java

```diff
@@ -14,9 +14,9 @@
  */
 @JsonDeserialize(using = SortClauseDeserializer.class)
 @Schema(
-    type = SchemaType.ARRAY,
-    implementation = String[].class,
+    type = SchemaType.OBJECT,
+    implementation = Object.class,
```

Comment:
Should this be `java.util.Map`? (JSON Object -> Java Map)

Reply:
OK will change, think need to fix it in FilterClause too.